### PR TITLE
Release version 1.0.0

### PR DESCRIPTION
**Pull request by aboubakar14**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
-## [Unreleased]
+## [1.0.0] - 2022/10/15
 
 ### Added
```
Comment (Zorin95670) [inline changelog.md:10]: missing break line

File: changelog.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
-## [Unreleased]
+## [1.0.0] - 2022/10/15
```
Comment (Zorin95670) [inline changelog.md]: Back to the future

File: changelog.md (hunk 1 above)
Comment (Zorin95670) [reply changelog.md:10]: Missing summary at the end like over changelog

File: src/main.py [hunk 3]
```diff
@@ -19,3 +19,10 @@ async def root() -> dict:
     This endpoint is the entrypoint of the API and returns a welcome message.
     """
     return {"message": "Hello From Leto-Modelizer-AI-API!"}
+
+@app.get("/health")
+async def root() -> dict:
```
Comment (Zorin95670) [inline src/main.py]: Must be in another PR

**Review by Zorin95670 — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (aboubakar14) [reply changelog.md:10]: done

File: changelog.md (hunk 2 above)
Comment (aboubakar14) [reply changelog.md]: done

File: src/main.py (hunk 3 above)
Comment (aboubakar14) [reply src/main.py]: done

**Review by Zorin95670 — APPROVED** [review]
